File: public/phase-2-20160620022203/JAVA/trains.java
// Name : Trains
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n : number of cities
		// m : number of  roads connecting the cities
		// k : number of railways
		// uvx : uvx corresponds to ui vi xi
		// sy : uvx corresponds to si yi
		int idx, jdx;
		Scanner in = new Scanner(System.in);
		String[] elements = (in.nextLine()).split(" ");
		int n = Integer.parseInt(elements[0]);
		int m = Integer.parseInt(elements[1]);
		int k = Integer.parseInt(elements[2]);
		int[][] uvx = new int[m][3];
		String[] uvx_elements = new String[100];
		for(idx=0;idx<m;idx++){
			uvx_elements = (in.nextLine()).split(" ");
			for(jdx=0;jdx<3;jdx++){
				uvx[idx][jdx] = Integer.parseInt(uvx_elements[jdx]);
			}
		}
		int[][] sy = new int[k][2];
		String[] sy_elements = new String[100];
		for(idx=0;idx<k;idx++){
			sy_elements = (in.nextLine()).split(" ");
			for(jdx=0;jdx<2;jdx++){
				sy[idx][jdx] = Integer.parseInt(sy_elements[jdx]);
			}
		}


		// write your code here
		// store your results in `result`

		// output
		// Dummy Data
		int result=2;

		System.out.println(result);

	}
}

File: public/phase-2-20160620022203/JAVASCRIPT/trains.js
// Name : Trains
// input:
// n : number of cities
// m : number of  roads connecting the cities
// k : number of railways
// uvx : uvx corresponds to ui vi xi
// sy : uvx corresponds to si yi
var idx,result;
var elements = readline().split(" ");
var n = parseInt(elements[0], 10);
var m = parseInt(elements[1], 10);
var k = parseInt(elements[2], 10);
var uvx = new Array([]);
var uvx_elements = "";
for(idx=0;idx<m;idx++){
	uvx_elements = readline().split(' ');
	uvx[idx] = new Array([]);
	for(jdx=0;jdx<3;jdx++){
		uvx[idx][jdx] = parseInt(uvx_elements[jdx], 10);
	}
}var sy = new Array([]);
var sy_elements = "";
for(idx=0;idx<k;idx++){
	sy_elements = readline().split(' ');
	sy[idx] = new Array([]);
	for(jdx=0;jdx<2;jdx++){
		sy[idx][jdx] = parseInt(sy_elements[jdx], 10);
	}
}

// write your code here
// store your results in `result`

// Dummy Data
result = 2;


// output
print(result);
